sympy/core/mul.py

```diff
@@ -1401,6 +1401,9 @@ def _eval_pos_neg(self, sign):
                 saw_NON = True
             elif t.is_extended_nonnegative:
                 saw_NON = True
+            # FIXME: is_positive/is_negative is False doesn't take account of
```

Comment:
Running this on 2.7 I get
```
SyntaxError: Non-ASCII character '\xc2' in file /home/jks/test/sympy/sympy/core/mul.py on line 1404, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```
The character between '#` and 'FIXME' seems to be a no-break space.